robottelo/bz_helpers.py

```diff
@@ -42,10 +46,19 @@ def get_decorated_bugs():  # pragma: no cover
     return bugs
 
 
-def get_deselect_bug_ids(bugs=None, log=None):
+def get_deselect_bug_ids(bugs=None, log=None):  # pragma: no cover
```

Comment:
This helper fetches BugZilla API and we are not going to fetch it during travis tests, so I added `pragma: no cover` and if in future we mock bugzilla API then we can remove it and add coverage.